docs/week-3.md

```diff
@@ -0,0 +1,523 @@
+
```

Comment:
can you please add a small agenda so I can put it in our events document, day view: https://docs.google.com/document/d/1CbTWfAhkmy7l-BbuaZJ1lSIsheUOcVfQRaBQPGAZRH4/edit?usp=sharing